html/elements/input/password.json

```diff
@@ -0,0 +1,1070 @@
+              "support": {
+                "chrome": {
+                  "version_added": "5",
+                  "notes": "Chrome 10 added support for <code><a href='https://developer.mozilla.org/docs/Web/HTML/Element/select'><select></a></code> element."
+                },
+                "chrome_android": {
+                  "version_added": null
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "4"
+                },
+                "firefox_android": {
+                  "version_added": true
+                },
+                "ie": {
+                  "version_added": "10"
+                },
+                "opera": {
+                  "version_added": "10"
+                },
+                "opera_android": {
+                  "version_added": true
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": null
+                },
+                "webview_android": {
+                  "version_added": null
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "size": {
+            "__compat": {
+              "description": "<code>size</code> attribute",
+              "support": {
+                "chrome": {
+                  "version_added": "1"
+                },
+                "chrome_android": {
+                  "version_added": true
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "1"
+                },
+                "firefox_android": {
+                  "version_added": "4"
+                },
+                "ie": {
+                  "version_added": "2"
+                },
+                "opera": {
+                  "version_added": "2"
+                },
+                "opera_android": {
+                  "version_added": true
+                },
+                "safari": {
+                  "version_added": "1"
+                },
+                "safari_ios": {
+                  "version_added": true
+                },
+                "webview_android": {
+                  "version_added": null
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "crossed-lock": {
```

Comment:
I'm not sure this is appropriate to include in the data for the password input. It is sort of relevant but only indirectly. It seems more like a generic browser UI decision rather than some aspect of the input itself.